build.sbt

```diff
@@ -271,6 +280,23 @@ lazy val connectCommon = (project in file("spark-connect/common"))
       PB.gens.java -> (Compile / sourceManaged).value,
       PB.gens.plugin("grpc-java") -> (Compile / sourceManaged).value
     ),
+    Compile / packageBin := assembly.value,
+    (assembly / test) := { },
```

Comment:
Basically just following other examples in this Delta's `build.sbt` file

https://github.com/delta-io/delta/blob/93eef1112fce9c766aec504f26f09d53bbcabb03/build.sbt#L47

Reply:
This empty block `{ }` indicates we're not testing this correct?
If so please add a `// TODO` inside the block.

Reply:
Hey @scottsand-db, I just have a quick question. 

Why did we always do `(assembly / test) := { }` for other projects in [build.sbt](https://github.com/delta-io/delta/blame/cddde68533f243ab049aaec4e464cf4e35a4db72/build.sbt#L47), so basically no testing task for `assembly`? Is it because we don't need to do it (like QA-ing near a Delta Release is good enough)?

https://github.com/delta-io/delta/commit/60a582b51fff28aa73b34153b3450c136e1ce96b#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91